.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+ - [ ] Defect
+ - [ ] Feature Request or Change Proposal
+ 
+## Defects 
+
+Make sure that these boxes are checked before submitting your issue -- thank you!
+
+ - [ ] Included `gnatsd -DV` output
+ - [ ] Included a [Minimal, Complete, and Verifiable example] (http://stackoverflow.com/help/mcve)
+ 
+#### Versions of `gnatsd` and affected client libraries used:
+
+#### OS/Container environment:
+
+#### Steps or code to reproduce the issue:
+ 
+#### Expected result:
+ 
+#### Actual result:
+ 
+ 
+## Feature Requests
+ 
+#### Use Case:
+ 
+#### Proposed Change:
+ 
+#### Who Benefits From The Change(s)?
+ 
+#### Alternative Approaches You've Ruled Out
```

Comment:
Good point. The fact that they are *alternative* approaches means they're not preferred, by definition.